src/test/java/com/julienviet/pgclient/PgConnectionTestBase.java

```diff
@@ -1060,6 +920,44 @@ public void testSetAndGetSerializableTx(TestContext ctx) {
     });
   }
 
+  @Test
+  public void testPgUpdate(TestContext ctx) {
+    Async async = ctx.async();
+    PgClient client = PgClient.create(vertx, options);
+    connector.accept(client, ctx.asyncAssertSuccess(conn -> {
+      PgPreparedStatement ps = conn.prepare("UPDATE Fortune SET message = 'PgClient Rocks!' WHERE id = 2");
+      PgUpdate update = ps.update();
+      update.execute(ctx.asyncAssertSuccess(result -> {
+        ctx.assertEquals(1, result.getUpdated());
+        conn.prepare("SELECT message FROM Fortune WHERE id = 2")
+          .query()
+          .execute(ctx.asyncAssertSuccess(r -> {
+            ctx.assertEquals("PgClient Rocks!", r.getRows().get(0).getValue("message"));
+            async.complete();
+          }));
+      }));
+    }));
+  }
+
+  @Test
+  public void testPgUpdateWithParams(TestContext ctx) {
```

Comment:
testUpdateWithParams

Reply:
This will be renamed to testPreparedUpdateWithParams